tests/acceptance/pageobjects/facetspage.js

```diff
@@ -0,0 +1,38 @@
+import SearchComponentBlock from '../blocks/searchcomponent';
+import VerticalResultsComponentBlock from '../blocks/verticalresultscomponent';
+import FacetsComponentBlock from '../blocks/facetscomponent';
+
+/**
+ * A model of a universal search page, containing block representations
```

Comment:
A model of a vertical search page with facets...

Reply:
done